tests/api/v1/subjects/put.js

```diff
@@ -637,6 +637,33 @@ describe('tests/api/v1/subjects/put.js >', () => {
         done();
       });
     });
+
+    it('tags and related links not provided, should set to empty array',
+      (done) => {
```

Comment:
Are we already testing this for all other endpoints?

Reply:
not explicitly checking tags and related links in other possible tests. Added now.